src/sinks/gcp/stackdriver_logs.rs

```diff
@@ -172,8 +171,11 @@ impl HttpSink for StackdriverSink {
             .map(remap_severity)
             .unwrap_or_else(|| 0.into());
 
+        let mut event = Event::Log(log);
+        self.config.encoding.apply_rules(&mut event);
```

Comment:
Do you think we should open an issue to make `apply_rules` take a `LogEvent`?

Reply:
Or the reverse, `fn LogEvent::apply_encoding(&mut self, encoding: &Encoding)`?

Reply:
Yeah, it would be nice to have any of these